modules/util/portal-tools-db-upgrade-client/src/testFunctional/tests/DatabaseUpgradeClient.testcase

```diff
@@ -96,6 +96,7 @@ definition {
 	@description = "LPS-158522: Upgrade client should not be able to run on versions greater than Java SE 8 "
 	@priority = 3
 	test CheckUpgradeClientJavaVersion {
+		property portal.upstream = "quarantine";
```

Comment:
@joshitagaki This is under "Database Upgrade Tool" component, need to notify Database Infra Team that you are adding this test to quarantine. Their team seems to already has the test suite that will be able to catch quarantine.

Reply:
@julschong just wanted to confirm was this the only test to be sent to another team?